clients/web/src/lib/components/molecules/user-menu/UserMenu.svelte

```diff
@@ -19,6 +19,10 @@
             label: "Sign Out",
             action: async () => goto("/auth/logout"),
         },
+        {
+            label: "Switch Organization",
+            action: async () => goto("/auth/org"),
+        },
```

Comment:
If the user has multiple organizations, show this; otherwise it should not be shown